src/QMCWaveFunctions/SPOSetBuilderFactory.cpp

```diff
@@ -246,10 +247,12 @@ SPOSet* SPOSetBuilderFactory::createSPOSet(xmlNodePtr cur)
   std::string bname("");
   std::string sname("");
   std::string type("");
+  std::string rotation("no");
   OhmmsAttributeSet aAttrib;
   aAttrib.add(bname, "basisset");
   aAttrib.add(sname, "name");
   aAttrib.add(type, "type");
+  aAttrib.add(rotation, "optimize");
```

Comment:
To be more precise, orbital optimization is also possible without using rotation. We may need more precise input paramters. I'm thinking of optimize="rotate" instead of just optimize="yes". To be discussed.